session.go

```diff
@@ -23,8 +24,10 @@ type session struct {
 	net.Conn
 	windowSize    int
 	maxPadding    *big.Int
```

Comment:
I just felt it's unnecessary to convert the type to and from `big.Int`. `rand.Int` is called only when `needsPadding` (which is calculated from the int `maxPadding`) is true, and the cost of `big.NewInt` seems neglectable comparing to zeroing the padding memory.